src/SourceBuild/content/eng/finish-source-only.proj

```diff
@@ -114,12 +114,22 @@
           Outputs="$(BaseIntermediateOutputPath)ReportPoisonUsage.complete" >
     <ItemGroup>
       <!-- Exclude the Private.SourceBuilt.Artifacts archive from poison usage scan. -->
-      <PoisonFileToCheck Include="$(ArtifactsAssetsDir)*$(ArchiveExtension)" />
-      <PoisonFileToCheck Remove="$(ArtifactsAssetsDir)$(SourceBuiltArtifactsTarballName)*" />
+      <AssetToCheck Include="$(ArtifactsAssetsDir)*$(ArchiveExtension)" />
+      <AssetToCheck Remove="$(ArtifactsAssetsDir)$(SourceBuiltArtifactsTarballName)*" />
       <!-- Include shipping nuget packages. -->
-      <PoisonFileToCheck Include="$(ArtifactsShippingPackagesDir)*.nupkg" />
+      <ShippingPackageToCheck Include="$(ArtifactsShippingPackagesDir)**/*.nupkg" />
```

Comment:
super tiny nit: inconsistent use path separator char. L122 uses `\` which I would go with.

Reply:
This whole targets file is Linux specific, so `/` isn't out of place, i.e. https://github.com/dotnet/sdk/blob/5ff7ddace486d6bc80d9cbc1a3a2a2910da0cc19/src/SourceBuild/content/eng/finish-source-only.proj#L56